channels/tw.m3u

```diff
@@ -141,3 +141,5 @@ https://lylive-videorent.cdn.hinet.net/out/u/live/gop4/ly/ly-Live1/hls-cl-tv/ind
 http://118.163.88.61:1935/hwazanlivetv/live.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="HuaShiXingWen.tw" tvg-name="華視新聞資訊" tvg-country="TW" tvg-language="Chinese" tvg-logo="https://upload.wikimedia.org/wikipedia/zh/thumb/2/27/Cts_news_info.png/200px-Cts_news_info.png" group-title="News",華視新聞資訊
 http://seb.sason.top/sc/hsxw_fhd.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="TW" tvg-language="Chinese" tvg-logo="" group-title="",CTI TV Asia
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/channel/UC5l1Yto5oOIgRXlI4p4VKbw/live
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/120884274-e6323000-c5ea-11eb-9d49-cca75e572c01.png)

Reply:
My streamlink doesn't seem to understand some of these links.